ci/Dockerfile.ci

```diff
@@ -0,0 +1,112 @@
+################################################################################
+# Build Stage 1: KubeVirt Test Binary
+################################################################################
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.6 AS kubevirt-builder
+
+ARG KUBEVIRT_VERSION=v1.5.0
```

Comment:
is it possible to make the kubevirt (and SSP) versions dynamic? kubevirt v1.5.0 corresponds to CNV 4.19.
and we might want to take the latest z-stream available (for 1.5, there's v1.5.3 already)

Reply:
Right, I was concerned about that too, so I decided to force the Dockerfile to never be able to run any tests: https://github.com/openshift-cnv/ocp-virt-validation-checkup/pull/14/files#diff-240d32f4199b7fc54ff5eded9116731629885deaf20a3856eec46e743698fa11R96

The job using this `Dockerfile.ci` will only run with `DRY_RUN=true`, meaning that the content of the tests is not important. To provide stability, a known stable version is used here.

What do you think about this strategy? Are there any edge cases I might be missing?